### ISSUE TITLE
Does this lib have Typings?

### ISSUE THREAD

**Issue by distante** [instructure/pdf-annotate.js#35]
Is there Typings available for pdf-annotate.js ? I would like to use it in a Angular4 /Typescript based project.

Thanks in advance.

**Comment by donniekerr** [comment]
Did you get this working in Angular?  I got it to load the pdf but not the annotator UI at top.

**Comment by distante** [comment]
No, we ended up making out our own implementation of the annotator.
